starboard/evergreen/shared/launcher.py

```diff
@@ -271,6 +275,43 @@ def _StageTargetsAndContentsRaspi(self):
     os.makedirs(os.path.join(target_staging_dir, 'lib'))
     shutil.copy(target_binary_src, target_binary_dst)
 
+  def _StageTargetsAndContentsRdk(self):
+    """Stage targets and their contents for GN builds for RDK platforms."""
+    content_subdir = os.path.join('usr', 'share', 'cobalt')
+
+    # TODO(b/296633713):
+    # Target_name should be self.target_name. However because loader is
+    # hardcoded on RDK, i.e. it always loads "libcobalt.so" in path "cobalt",
+    # it has to use this name.
+    target_name = 'cobalt'
```

Comment:
Is there a reason to add this instead of just using `-t cobalt` and having other `-t` values fail?

Reply:
Looks like I misunderstood, once comment is updated lgtm